hudi-client/hudi-client-common/src/main/java/org/apache/hudi/keygen/KeyGenUtils.java

```diff
@@ -41,6 +41,7 @@ public class KeyGenUtils {
   protected static final String EMPTY_RECORDKEY_PLACEHOLDER = "__empty__";
 
   protected static final String HUDI_DEFAULT_PARTITION_PATH = PartitionPathEncodeUtils.DEFAULT_PARTITION_PATH;
+  public static final String OLD_DEPRECATED_DEFAULT_PARTITION_PATH = "default";
```

Comment:
let's rename it `DEPRECATED_DEFAULT_PARTITION_PATH`? Also, should we move this constant to `PartitionPathEncodeUtils`? That's where the old one was defined.